src/eredis_client.erl

```diff
@@ -372,7 +375,7 @@ connect_next_addr([Addr|Addrs], Port, SocketOptions, ConnectOptions, State) ->
                                     {ok, State#state{socket = NewSocket}};
                                 {error, Reason} ->
                                     close_socket(State, NewSocket),
-                                    {error, {select_error, Reason}}
+                                    {error, {select_db_error, Reason}}
```

Comment:
Why change this? Maybe someone relies on this atom...

Reply:
Just niceness, but you are right, Ill revert that